### PR TITLE
[13.0][ADD] account_analytic_tag_default

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Set default default tags on analytic accounts that apply to invoices when no
other tag is specified and when changing the analytic account.

@ForgeFlow

### CODE REVIEW

File: account_analytic_tag_default/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Analytic Tag Default",
+    "summary": "Set default tags on analytic accounts.",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://www.github.com/OCA/account-analytic.git",
+    "depends": ["account", "analytic"],
```
Comment (MiquelRForgeFlow) [inline account_analytic_tag_default/__manifest__.py]:
```suggestion
    "depends": ["account"],
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional + code. Nice one!

**Comment by LoisRForgeFlow** [comment]
Squashed commits, ready to merge IMO.

**Pull request merged**